src/commands/codepush/deployment/list.ts

```diff
@@ -34,4 +50,76 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
       }
     }
   }
+
+  private generateColoredTableTitles(tableTitles: string[]): string[] {
+    return tableTitles.map((title) => chalk.cyan(title));
+  }
+
+  private async generateTableInfoRows(deployments: models.Deployment[], client: AppCenterClient): Promise<string[][]> {
+    return await Promise.all(deployments.map(async (deployment: models.Deployment): Promise<string[]> => {
+      let metadataString: string = "";
+      let metricsString: string = "";
+
+      if (deployment.latestRelease) {
+        metadataString = this.generateMetadataString(deployment.latestRelease);
+
+        let metrics: models.CodePushReleaseMetric[];
+        const httpRequest = await out.progress("Getting CodePush deployments metrics...", clientRequest<models.CodePushReleaseMetric[]>(
+          (cb) => client.codePushDeploymentMetrics.get(deployment.name, this.app.ownerName, this.app.appName, cb)));
+        metrics = httpRequest.result;
+
+        let releasesTotalActive: number = 0;
+        metrics.forEach((metric) => releasesTotalActive += metric.active);
+
+        const releaseMetrics: models.CodePushReleaseMetric = metrics.find((metric) => metric.label === deployment.latestRelease.label);
+        if (releaseMetrics) {
+          metricsString = this.generateMetricsString(releaseMetrics, releasesTotalActive);
+        } else {
+          metricsString = chalk.magenta("No installs recorded");
+        }
+      } else {
+        metadataString = chalk.magenta("No updates released");
+        metricsString = chalk.magenta("No installs recorded");
+      }
+
+      return [deployment.name, metadataString, metricsString];
+    }));
+  }
+
+  private generateMetricsString(releaseMetrics: models.CodePushReleaseMetric, releasesTotalActive: number): string {
+    let metricsString: string = "";
+
+    const activePercent: number = (releaseMetrics.downloaded) ? releaseMetrics.active / releasesTotalActive * 100 : 0.0;
```

Comment:
Although it looks like there are no ways for `releasesTotalActive` to be `0` here - could we add protective code in case the calling code changes (so we don't get a divide by zero error here).

Reply:
Yeah, you are absolutely right. Thanks. Fixed.